src/software/simulated_tests/validation_functions/non_keeper_touched_ball_in_friendly_defense_validation.cpp

```diff
@@ -0,0 +1,41 @@
+#include "software/new_geom/util/contains.h"
+#include "software/new_geom/util/intersects.h"
+#include "software/simulated_tests/validation_functions/touch_ball_in_enemy_defense_validation.h"
+
+void nonKeeperTouchedBallInFriendlyDefense(std::shared_ptr<World> world_ptr,
+                                           ValidationCoroutine::push_type &yield)
+{
+    auto robot_touched_ball = [](std::shared_ptr<World> world_ptr) {
+        Rectangle friendly_defense_area = world_ptr->field().friendlyDefenseArea();
+
+        Point ball_position = world_ptr->ball().position();
+        Circle ball_aabb    = Circle(ball_position, BALL_MAX_RADIUS_METERS);
```

Comment:
There are a couple strange '_aabb' variable names here.

I assume that's a bad refactor?